.travis.yml

```diff
@@ -1,26 +1,45 @@
-sudo: enabled
-
+dist: xenial
+sudo: required
 language: c
 
 install:
-    - make build-depends
-    - sudo apt-get -y install debian-archive-keyring dpkg=1.17.5ubuntu5.8
+  - make build-depends
+  - sudo apt-get -y install debian-archive-keyring
 
 # See the Ubuntu section in the debian/README file for more details, but this
 # is a good example of why CI is useful /before/ shipping a product..
 #
 # Also, the age of the debian-archive-keyring (2012) is simply concerning. So,
 # I have to manually add the debian stable release keys
 before_script:
-    - sudo sed -i '989s/$forceyes//' /usr/sbin/multistrap
-    - sudo apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 7638D0442B90D010
-    - sudo apt-key adv --keyserver keyserver.ubuntu.com --recv-keys EF0F382A1A7B6500
+  - sudo apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 7638D0442B90D010
+  - sudo apt-key adv --keyserver keyserver.ubuntu.com --recv-keys EF0F382A1A7B6500
```

Comment:
Did you test if the keys still needed to be added in xenial?

Reply:
These are no longer needed. I will remove them.

Reply:
Well nevermind, works on my local xenial VM but not travis?
https://travis-ci.org/tomeshnet/mesh-orange/builds/468504273#L1095

Reply:
Okay now I am completely lost, it is failing even after I reverted the commit. It's a new problem since yesterday when the build succeeded?
https://travis-ci.org/tomeshnet/mesh-orange/jobs/468507577#L1113

Reply:
That was a temporary debian url failure, the keys are _not_ needed.